libs/application/templates/parental-leave/src/fields/Rights/SummaryRights.tsx

```diff
@@ -68,7 +68,7 @@ export const SummaryRights = ({ application }: SummaryRightsProps) => {
               </>
             )}
 
-            {isGivingRights === YES && giveDays > 0 && (
```

Comment:
Should be back to `>` right?

Reply:
`const { giveDays} = useApplicationAnswers(application)` returns a negative value in this scenario:
1. Primary parent applies first
2. Secondary parent approves giving the days
3. Secondary parent applies and reaches the Summary screen.

I think we set it to a negative value because VMLST requested it that way...

So the question is, is the negative value a bug? If not, then we need to keep it `< 0` to show this block...

What do you think?

Reply:
Ah yeah, I read it too fast last time. I thought it was using the `given` variable instead, which now use `Math.abs` hence my comment above.

I believe we need another case then. If the primary parent decides to gives his days, the value will be positive, right? We might just want to check if `giveDays !== 0`

Reply:
Sounds good, thanks for taking another look. I've changed it to `giveDays !== 0`. I was doing some testing around it, and have a hard time reaching the give rights screen on dev with mock data (but that's a separate issue that I'll continue to investigate for a future PR).